tests/tools_tests/setup_test_workspace.sh

```diff
@@ -48,6 +48,9 @@ bazelrc_template="
 build --deleted_packages=
 query --deleted_packages=
 # EOF"
+# todo: should these lines be in the template above?
+# startup --windows_enable_symlinks
+# common --enable_runfiles
```

Comment:
I presume that we will determine the answer before we merge this PR.

Reply:
Yes - can you help me understand that template? Is it used only for test cases internal to this repo or tests outside? Do you have a view as to whether these flags should be in here?

Reply:
I've removed this